File: _CodeTopics/LeetCode/1-200/000088/000088.py
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """

        # 今天是清明节假期看媳妇儿从西安返程（从洛阳龙门站回来的基本。。。），到住的地方没多久，赶紧搞一下。
        
        for i in range(-1, -n-1, -1):
            nums1[i] = nums2[i]
        nums1.sort()
        
"""
https://leetcode-cn.com/submissions/detail/164116933/

59 / 59 个通过测试用例
状态：通过
执行用时: 20 ms
内存消耗: 12.9 MB

执行用时：20 ms, 在所有 Python 提交中击败了64.43%的用户
内存消耗：12.9 MB, 在所有 Python 提交中击败了73.66%的用户
"""

File: _CodeTopics/LeetCode/1-200/000088/tran/000088.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {

        // https://leetcode.cn/problems/merge-sorted-array/submissions/164116933/
        for (int i = 0; i < n; ++i) {
            nums1[i+m] = nums2[i];
        }
        std::sort(nums1.begin(), nums1.end());
    }
};
/*
https://leetcode.cn/problems/merge-sorted-array/submissions/455944471/

时间
详情
-ms
击败 100.00%使用 C++ 的用户
内存
详情
8.61mb
击败 48.88%使用 C++ 的用户
*/
/*
现在时间太短的话不是 0ms 了，是直接不显示时间了。
*/
